e2e/cypress/integration/channel/channel_info_rhs_spec.js

```diff
@@ -0,0 +1,395 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
@JulienTant Since this is a new E2E it needs a test case number. You can use MM-T4685 here.